Pipfile

```diff
@@ -13,6 +13,7 @@ pre-commit = "*"
 aws-sam-cli = "*"
 boto3 = "*"
 botocore = "*"
+jsondiff = "*"
```

Comment:
all dependencies should contain versions, like in our standard template https://github.com/Sage-Bionetworks-IT/lambda-template/blob/master/Pipfile

you should also commit the corresponding `Pipfile.lock` file.  Run `pipenv install --dev` to get an updated Pipfile.lock then commit that.